emData/download.sh

```diff
@@ -78,8 +78,418 @@ declare -a processing_modules=(
   "VMRCM_D5PHIA"
 
   # TrackletEngine
+  "TE_L1PHIA1_L2PHIA1"
```

Comment:
Maybe not for this PR, but the download script is starting to get a bit long. Since we're starting to test more modules, maybe we should just handle all processing modules automatically and not have to specify each one explicitly. (I sure hope you didn't do this by hand :-) )

Reply:
I used the magic of awk! (… and grep… and sed).

And yeah, I was thinking the same thing. I just pushed a commit that gets rid of the manual list and just creates links for all modules. It takes longer to run (~1.5 minutes in my testing), but I think we're at the point where having the test vectors for all the modules is a useful thing.

Let me know what you think.

Reply:
Running the download for 1.5 minutes once in awhile doesn't sound too bad. We don't do a fw synch often enough for that to be a pain in my opinion.